pkg/helper/bridges_test.go

```diff
@@ -0,0 +1,132 @@
+package helper
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	nmstatev1alpha1 "github.com/nmstate/kubernetes-nmstate/pkg/apis/nmstate/v1alpha1"
+)
+
+var (
+	badYaml = nmstatev1alpha1.State("}")
+	empty   = nmstatev1alpha1.State("")
+
+	noBridges = nmstatev1alpha1.State(`interfaces:
+  - name: eth1
+    type: ethernet
+    state: up
+  - name: bond1
+    type: bond
+    state: up
+    link-aggregation:
+      mode: active-backup
+      slaves:
+        - eth1
+      options:
+        miimon: '120'
+`)
+	noBridgesUp = nmstatev1alpha1.State(`interfaces:
+  - name: eth1
+    type: ethernet
+    state: up
+  - name: br1
+    type: linux-bridge
+    state: down
```

Comment:
Is state mandatory? If not, what is the default?

Reply:
I think it's mandatory https://github.com/nmstate/nmstate/blob/master/libnmstate/schemas/operational-state.yaml#L71 it does not have minimun: 0